Formula/c/cfn-lint.rb

```diff
@@ -3,10 +3,15 @@ class CfnLint < Formula
 
   desc "Validate CloudFormation templates against the CloudFormation spec"
   homepage "https://github.com/aws-cloudformation/cfn-lint/"
-  url "https://files.pythonhosted.org/packages/42/dc/c6afb068eddaba2ba462a0c62645484f6b0d0bc77859c16339f1ba8a6718/cfn-lint-0.86.2.tar.gz"
-  sha256 "c140c814b6a7db048ac881a4f9b7f4e8335afd706f523f3fbf884eb6322f16f9"
+  url "https://files.pythonhosted.org/packages/c9/6d/5cfcf9e4cf5cab340b623cc988051e5bbe86034a6f7ad91a0ab7c4e5f523/cfn_lint-0.86.3.tar.gz"
+  sha256 "044c929557cd45cab3005055fa0e7e70997d8dcfd2217e75f9ecfd6b328ac8eb"
   license "MIT-0"
 
+  livecheck do
+    url "https://pypi.org/rss/project/cfn-lint/releases.xml"
+    regex(%r{<title>v?(\d+(?:\.\d+)+)</title>}i)
```

Comment:
```suggestion
    url :stable
    regex(/href=.*?cfn_lint[._-]v?(\d+(?:\.\d+)+)\.t/i)
```

The reason why the `Pypi` strategy doesn't successfully return versions by default and this falls back to checking the `Git` strategy (where the `1.2.5.a5` version comes from) is because the generated strategy regex uses `cfn-lint` but the name in the filename is `cfn_lint`. We can address this by simply adding an appropriate regex, so checking `releases.xml` isn't necessary here.